conf/app.src

```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://download.dokuwiki.org/src/dokuwiki/dokuwiki-2018-04-22b.tgz
-SOURCE_SUM=9d1437cdc7e98e471ff32079f7ca224ccf52dbbaafed39ec9746b0405b3a29ce
+SOURCE_URL=https://github.com/splitbrain/dokuwiki/archive/release_stable_2020-07-29.tar.gz
```

Comment:
Is there any reason for not sticking to the author website ?

Reply:
I guess both work. it is just that GitHub repository gives info like changes log, update alert, issues... and it is perhaps less disturbing for someone who would like to do a PR DokuWiki upgrade.

Reply:
You're right. GitHub is more "appealing and widespread"